locale/tr/docs/es6.md

```diff
@@ -0,0 +1,40 @@
+---
+title: ECMAScript 2015 (ES6) ve ötesi
+layout: docs.hbs
+---
+
+# ECMAScript 2015 (ES6) ve ötesi
+
+Node.js [V8](https://v8.dev/)'in modern sürümlerine göre oluşturulmuştur. Bu motorun en son sürümlerini güncel tutarak, [JavaScript ECMA-262 spesifikasyonundaki](http://www.ecma-international.org/publications/standards/Ecma-262.htm) yeni özelliklerin zamanında Node.js geliştiricilerine getirilmesinin yanı sıra sürekli performans ve kararlılık iyileştirmeleri sağlıyoruz.
```

Comment:
* `Node.js geliştiricilerine getirilmesinin yanı sıra ...`
"getirmek" sounds not like a "feature" here
**Suggestion:** So maybe referring to `ulastirilmasiyla` or `saglanmasiyla` referring to "deliver/ship". Probably `saglanmasiyla` might collide with the verb at the end, too. Wdyt? 🤔

Reply:
We can say: "...yeni özelliklerin zamanında Node.js geliştiricilerine ulaştırılmasının yanı sıra..."